python/lsst/ip/isr/isrFunctions.py

```diff
@@ -347,6 +349,92 @@ def maskITLEdgeBleed(ccdExposure, badAmpDict, itlEdgeBleedSatMinArea=10000,
                         sliceMask[y, lowerRange:upperRange] = satMaskBit
 
 
+def maskITLDip(exposure, detectorConfig, maskPlaneNames=["SUSPECT", "ITL_DIP"], log=None):
+    """Add mask bits according to the ITL dip model.
+
+    Parameters
+    ----------
+    exposure : `lsst.afw.image.Exposure`
+        Exposure to do ITL dip masking.
+    detectorConfig : `lsst.ip.isr.overscanAmpConfig.OverscanDetectorConfig`
+        Configuration for this detector.
+    maskPlaneNames : `list [`str`], optional
+        Name of the ITL Dip mask planes.
+    log : `logging.Logger`, optional
+        If not set, a default logger will be used.
+    """
+    if detectorConfig.itlDipBackgroundFraction == 0.0:
+        # Nothing to do.
+        return
+
+    if log is None:
+        log = logging.getLogger(__name__)
+
+    thresh = afwDetection.Threshold(
+        exposure.mask.getPlaneBitMask("SAT"),
+        afwDetection.Threshold.BITMASK,
+    )
+    fpList = afwDetection.FootprintSet(exposure.mask, thresh).getFootprints()
+
+    heights = numpy.asarray([fp.getBBox().getHeight() for fp in fpList])
+
+    largeHeights, = numpy.where(heights >= detectorConfig.itlDipMinHeight)
+
+    if len(largeHeights) == 0:
+        return
+
+    # Get the approximate image background.
+    approxBackground = numpy.median(exposure.image.array)
+    maskValue = 0
+    for maskPlaneName in maskPlaneNames:
+        maskValue |= exposure.mask.getPlaneBitMask(maskPlaneName)
```

Comment:
You should be able to get the complete bitmask by `maskValue = exposure.mask.getPlaneBitMask(maskPlaneNames)` (example: https://github.com/lsst/cp_verify/blob/8f5961f9077d81a93a6cad45f2fd3402cd469aaf/python/lsst/cp/verify/verifyStats.py#L337)